packages/@guardian/source-react-components/src/button/styles.ts

```diff
@@ -100,23 +100,12 @@ const subdued = (
 	border-radius: 0;
 `;
 
-/*
```

Comment:
This was correcting the position of the text, setting the line-height to 1.35 rather than 1.5 removes the need for this code.